bots/lemmy_mlb_game_threads/__init__.py

```diff
@@ -6339,3 +6324,13 @@ def bot_state(self):
 
         self.log.debug("Bot Status: {}".format(botStatus))  # debug
         self.bot.detailedState = botStatus
+
+    def _truncate_post(self, text):
+        warning_text = '\n# Truncated: 10k Limit Reached'
```

Comment:
I used `warning_text = " \  # Truncated..."`. A back slash in markdown will produce a new line. Will \n work?

Reply:
It seems to be working, but I'm fine with using the other method.